docs/dashboards/synchronize-components.md

```diff
@@ -81,3 +81,77 @@ Dashboards.board('container', {
 |[Highcharts](https://www.highcharts.com/docs/dashboards/highcharts-component)|yes|yes|yes
 |[DataGrid](https://www.highcharts.com/docs/dashboards/datagrid-component)|yes|yes|yes
 |[KPI](https://www.highcharts.com/docs/dashboards/kpi-component)|no|yes|no
+
+### Creating a Custom Synchronization
```

Comment:
Here I would replace it with `##` and treat it as the main title for this section